app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] Foreign key already declared in the _User_ model, and hence no need to declare it again in a referenced model. Simply do something like:
```ruby
# app/models/like.rb
belongs_to :author, class_name: 'User'

Reply:
Thanks for pointing this out, I will make corrections now.

Reply:
- [ ] This comment has not been addressed yet